conda-forge.yml

```diff
@@ -1,4 +1,6 @@
 conda_forge_output_validation: true
+bot:
+  inspection: update-grayskull
```

Comment:
Any particular reason why this was added?

Reply:
I had noticed that there were quite a few conda-forge packages that I was encountering that had incorrect dependencies, and I asked if there was any way to try to address this issue.  I was informed that this can help keep recipes up-to-date.